src/mds/mdstypes.h

```diff
@@ -340,10 +340,10 @@ struct quota_info_t
   static void generate_test_instances(list<quota_info_t *>& ls);
 
   bool is_valid() const {
-    return max_bytes >=0 && max_files >=0;
+    return max_bytes >= 0 && max_files >= 0;
   }
   bool is_enable() const {
-    return max_bytes || max_files;
+    return max_bytes > 0 || max_files > 0 ;
```

Comment:
Github seems to have eaten a comment (which I saw by email) here:
```
@gregsfortytwo @jcsp
Not found the problem, just concerned the values could be negative,
and the same time, return a bool type better than int64_t type, because the is_enable()'return is bool type,and avoid the values could be negative.
```

So I don't think any of that is a reason to put the `> 0` here -- it would be reasonable to suggest that max_bytes should be unsigned, but given that it is signed, I don't think it is logical to interpret a "-1" value as making is_enable() return false.